Makefile

```diff
@@ -1,10 +1,10 @@
-.PHONY: help format lint mypy pytest test-all vulncheck
+.PHONY: help format lint mypy unittest test-all vulncheck
 
 help:
 	@echo "format    - Format python code with isort and black"
 	@echo "lint      - Check style with pylint"
 	@echo "mypy      - Run the static type checker"
-	@echo "pytest    - Run tests suite with pytest"
+	@echo "unittest  - Run tests suite with unittest"
```

Comment:
I am wondering if we shouldn't just call it `test`, I find it more common and it avoids being tied to a framework